src/App.js

```diff
@@ -11,7 +11,7 @@ class App extends Component {
 	render() {
   		return (
     		<div>
-				<Route path="/" exact render={(props) => <Home {...props}/>}/>
+				<Route path="/home" exact render={(props) => <Home {...props}/>}/>
```

Comment:
The path for the home page should be left as "/" because it'll show the home page as soon as we run npm start. Otherwise, it'll only show the home page when we type the url: localhost:3000/Home.